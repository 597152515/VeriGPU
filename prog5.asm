out abx
out cdx
out 12x
out 34x
li x1 01x
outr x1
li x1 02x
outr x1
li x1 dex
outr x1
li x1 adx
outr x1

20x:
short deadx
short beefx
