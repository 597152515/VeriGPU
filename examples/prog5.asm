out 0x1b
out 0x2d
out 0x32
out 0x44
li x1 0x01
outr x1
li x1 0x02
outr x1
li x1 0x1e
outr x1
li x1 0x2d
outr x1
li x21 22
li x23 33
outr x21
outr x23
halt

0x40:
word 0x0000dead
word 0x0000beef
