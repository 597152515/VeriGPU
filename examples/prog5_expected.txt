out 0 0000001b 27
out 1 0000002d 45
out 2 00000032 50
out 3 00000044 68
out 4 00000001 1
out 5 00000002 2
out 6 0000001e 30
out 7 0000002d 45
out 8 00000016 22
out 9 00000021 33